### PR TITLE
Calculate per-network traffic (#963)

### PR DESCRIPTION
**Pull request by jpnurmi**

### CODE REVIEW

**Comment by DarthGandalf** [comment]
No indication of it in existing UI?

**Comment by jpnurmi** [comment]
No, but [znc-admin](https://github.com/jpnurmi/znc-admin) (needs also #1019 and [341675c](https://github.com/jpnurmi/znc/commit/341675c230574e7d4c149a8ad14d8d4fc11fabc2)) can show per-network traffic

**Comment by DarthGandalf** [comment]
znc-admin isn't going to replace webadmin, right? Please add this to Traffic tab in webadmin.

**Comment by jpnurmi** [comment]
Ok, per-network traffic stats added to webadmin.

File: src/Client.cpp [hunk 1]
```diff
@@ -73,7 +73,10 @@ CClient::~CClient() {
 		CClientAuth* pAuth = (CClientAuth*) &(*m_spAuth);
 		pAuth->Invalidate();
 	}
-	if (m_pUser != nullptr) {
+	if (m_pNetwork != nullptr) {
```
Comment (jpnurmi) [inline src/Client.cpp]: btw, i'm not so sure if client side traffic should be associated to any network or not

File: include/znc/znc.h [hunk 2]
```diff
@@ -89,6 +89,7 @@ class CZNC {
 	// generated through ZNC.
 	TrafficStatsMap GetTrafficStats(TrafficStatsPair &Users,
 			TrafficStatsPair &ZNC, TrafficStatsPair &Total);
+	TrafficStatsPair GetNetworkTrafficStats(const CString& sUsername, TrafficStatsMap& Networks);
```
Comment (DarthGandalf) [inline include/znc/znc.h]: Any comments?
This function looks like a reverse of previous one (order of args/return)

Comment (jpnurmi) [inline include/znc/znc.h]: Ok. Changed to use the same order of args/return.

**Pull request merged**